docs/standards/universal-profile/lsp6-key-manager.md

```diff
@@ -84,10 +84,30 @@ Bear in mind that the behavior of `CHANGEPERMISSIONS` slightly varies depending
 
 </details>
 
+<details>
+    <summary><code>REENTRANCY</code> - Allows reentering during an execution</summary>
+    <p style={{marginBottom: '3%', marginTop: '2%', textAlign: 'center'}}>
+        <b>value = </b><code>0x0000000000000000000000000000000000000000000000000000000000000080</code>
+    </p>
+
+Given a contract has this permission, it enables that contract to be able to execute a payload during the execution of another payload.
```

Comment:
It might not be necessarily a contract, but also an EOA (signer) with executeRelayCall(...).
We might need to specify this.